variants/classicalcrowded/classicalcrowded.go

```diff
@@ -0,0 +1,470 @@
+package classicalcrowded
+
+import (
+	"github.com/zond/godip"
+	"github.com/zond/godip/graph"
+	"github.com/zond/godip/state"
+	"github.com/zond/godip/phase"
+	"github.com/zond/godip/variants/classical"
+	"github.com/zond/godip/variants/common"
+)
+
+const (
+	Balkans       godip.Nation = "Balkans"
+	Benelux       godip.Nation = "Benelux"
+	Iberia        godip.Nation = "Iberia"
+	Scandinavia   godip.Nation = "Scandinavia"
+)
+
+
+var (
+	Nations    = []godip.Nation{godip.Austria, Balkans, Benelux, godip.England, godip.France, godip.Germany, Iberia, godip.Italy, godip.Turkey, godip.Russia, Scandinavia}
+	SVGFlags = map[godip.Nation]func() ([]byte, error){
+		Balkans: func() ([]byte, error) {
+			return Asset("svg/balkans.svg")
+		},
+		Benelux: func() ([]byte, error) {
+			return Asset("svg/benelux.svg")
+		},
+		Iberia: func() ([]byte, error) {
+			return Asset("svg/iberia.svg")
+		},
+		Scandinavia: func() ([]byte, error) {
+			return Asset("svg/scandinavia.svg")
+		},
+		godip.Austria: func() ([]byte, error) {
+			return Asset("../classical/svg/austria.svg")
+		},
+		godip.England: func() ([]byte, error) {
+			return Asset("../classical/svg/england.svg")
+		},
+		godip.France: func() ([]byte, error) {
+			return Asset("../classical/svg/france.svg")
+		},
+		godip.Germany: func() ([]byte, error) {
+			return Asset("../classical/svg/germany.svg")
+		},
+		godip.Italy: func() ([]byte, error) {
+			return Asset("../classical/svg/italy.svg")
+		},
+		godip.Russia: func() ([]byte, error) {
+			return Asset("../classical/svg/russia.svg")
+		},
+		godip.Turkey: func() ([]byte, error) {
+			return Asset("../classical/svg/turkey.svg")
+		},
+	}
+	Parser = classical.Parser
```

Comment:
You can just write `classical.Parser` below - instead of declaring a new variable up here.